### PR TITLE
doc: advertise forum.riot-os.org

### PR DESCRIPTION
**Pull request by miri64**
### Contribution description
We have a forum now. We should advertise that everywhere where we also mention mailing lists.

### Testing procedure
Read.

### Issues/PRs references
None

### CODE REVIEW

**Comment by kfessel** [comment]
I was able to read it seems to be working as intended :)

File: Makefile [hunk 1]
```diff
@@ -38,7 +38,9 @@ welcome:
 	@echo ""
 	@echo "Please see our Quick Start Guide at:"
 	@echo "    https://doc.riot-os.org/getting-started.html"
-	@echo "Or ask questions on our mailing list:"
+	@echo "Or ask questions on our forum"
+	@echo "    https://forum.riot-os.org"
+	@echo "or mailing list:"
```
Comment (maribu) [inline Makefile:43]: For consistency, I would prefer both "or"s to either start with a capital or a lower case "o". Please squash right away.

**Review by maribu — APPROVED** [review]
ACK. If you agree on the comment, please squash right in.

File: Makefile (hunk 1 above)
Comment (miri64) [reply Makefile:43]: But it's the middle of the sentence. Now that I read it myself again, I am not happy about the double "or" though... Let me rephrase.

Comment (miri64) [reply Makefile:43]: Done and squashed.

**Review by maribu — APPROVED** [review]
ACK
